src/debug_info.rs

```diff
@@ -927,7 +952,16 @@ impl<'ctx> DebugInfoBuilder<'ctx> {
             )
         };
 
-        unsafe { InstructionValue::new(value_ref) }
+        #[cfg(any(feature = "llvm19-1", feature = "llvm20-1"))]
+        {
+            let _ = value_ref;
+            todo!()
```

Comment:
This looks like it still needs to be fixed? And a few later on as well

Reply:
I have no idea on how to use the new Debug Record API starting from LLVM 19, I think it is not just deprecated is it removed entirely? Or we can just version it out but it will be very confusing to end users

Reply:
Yeah I guess that new API deserve a whole new struct and APIs around it to be honest.